Packs/Cyberwatch/pack_metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Cyberwatch",
+    "description": "Find, prioritize, and fix vulnerabilities. Use this integration to fetch Assets, CVE, and Security issues from Cyberwatch.",
+    "support": "partner",
+    "created": "2024-07-01T10:00:00Z",
+    "currentVersion": "1.0.0",
+    "author": "Cyberwatch",
+    "url": "https://cyberwatch.fr",
+    "email": "support@cyberwatch.fr",
+    "categories": [
+        "Vulnerability Management"
+    ],
+    "tags": [],
+    "useCases": [
+        "Vulnerability Management"
+    ],
+    "keywords": [
+        "Cyberwatch",
+        "Framatome",
+        "Cyberwatch Vulnerability Manager",
+        "Cyberwatch Compliance Manager",
+        "Framatome Cybersecurity"
+    ],
+    "devEmail": [
+        "support@cyberwatch.fr"
+    ],
+    "githubUser": [
+        "tarraschk",
+        "fwininger"
+    ],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2",
+        "xsoar_on_prem",
+        "xsoar_saas"
```

Comment:
Some of these tags are irrelevant.
If you wish for this integration to run on all XSOAR versions (including XSIAM and XSOAR 8), then these two tags cover all the cases:
```suggestion
    "marketplaces": [
        "xsoar",
        "marketplacev2",
```

Reply:
👍 done